manim/mobject/opengl/opengl_mobject.py

```diff
@@ -2474,7 +2591,9 @@ def construct(self):
                 )
         return self
 
-    def pointwise_become_partial(self, mobject, a, b):
+    def pointwise_become_partial(
+        self, mobject: OpenGLMobject, a: float, b: float
```

Comment:
```suggestion
        self, mobject: OpenGLMobject, a: float, b: float, axis: Vector3D | None = None
```
To match the parent?

Reply:
This one is weird, because neither its counterpart `VMobject.pointwise_become_partial()` nor its overrides have that `axis` parameter, only `OpenGLSurface.pointwise_become_partial()` does.
So I didn't know what to do with this one, and I left it as is 🤷

Reply:
Interesting, it might be a good idea to make an issue and then resolve your conversation.

Reply:
Done! See #3805